packages/react-dev-utils/watchOptionsRegex.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+const path = require('path');
+const escapeStringRegexp = require('escape-string-regexp');
+
+module.exports = function watchOptionsRegex(paths) {
+  return new RegExp(
+    `^(?!${escapeStringRegexp(path.join(paths.appSrc, '/'))}).+node_modules`,
```

Comment:
This regex does not look identical to the old regex -- why the changes? They won't be functionally equivalent afaik.

Reply:
Since this its own package now, why don't we add a few test cases to check paths that this regex should be true/false against?

Reply:
@Timer I added the test cases, how should they be wired up in the CI scripts?

Reply:
Probably just call the tests in e2e simple similar to error overlay.